releases/pulpcore/settings

```diff
@@ -0,0 +1,5 @@
+OSES="el7 el8"
+KOJI_TAGS="pulpcore-@VERSION@-@OS@"
+PASS_NAME_GPG="theforeman/releases/pulpcore/gpg-passphrase"
+PASS_NAME_KEY="theforeman/releases/pulpcore/gpg-key"
+FULLGPGKEY=""
```

Comment:
this is intentional, as otherwise it wants to use the foreman key, but I don't want to be tight to foreman releases.